controllers/idler/idler_controller_test.go

```diff
@@ -1317,12 +1333,94 @@ func preparePayloadsSinglePod(t *testing.T, r *Reconciler, namespace, namePrefix
 	}
 }
 
-func createPods(t *testing.T, r *Reconciler, owner metav1.Object, startTime metav1.Time, podsToTrack []*corev1.Pod, conditions ...corev1.PodCondition) []*corev1.Pod {
+func preparePayloadCrashloopingPod(t *testing.T, r *Reconciler, namespace, namePrefix string) payloads {
```

Comment:
```suggestion
func preparePayloadCrashloopingAboveThreshold(t *testing.T, r *Reconciler, namespace, namePrefix string) payloads {
```

Reply:
done in [a147e63](https://github.com/codeready-toolchain/member-operator/pull/630/commits/a147e636496395427a349188b8cd8a91bfc020ea)